packages/react-network/src/server.ts

```diff
@@ -0,0 +1,74 @@
+import {Context} from 'koa';
+import {Header, StatusCode, CspDirective} from '@shopify/network';
+import {Manager} from './manager';
+
+export class ServerManager implements Manager {
+  private statusCodes: StatusCode[] = [];
+  private csp = new Map<CspDirective, string[] | boolean>();
+  private redirectUrl?: string;
+
+  redirectTo(url: string, status = StatusCode.Found) {
+    this.addStatusCode(status);
+    this.redirectUrl = url;
+  }
+
+  addStatusCode(statusCode: StatusCode) {
+    this.statusCodes.push(statusCode);
+  }
+
+  addCspDirective(directive: CspDirective, value: string | string[] | boolean) {
+    const normalizedValue = typeof value === 'string' ? [value] : value;
+    const newValue = Array.isArray(value)
```

Comment:
Don't think this check is necessary; between the typing and the line above `value` will always be an Array no?

Reply:
No, `value` can still be `boolean` here.